l16/teku/.env

```diff
@@ -0,0 +1,26 @@
+##############################################################################################################################
```

Comment:
It feels like this doesn't need to be here, right? we have the default.env already in the docker folder?